src/Og.php

```diff
@@ -273,4 +273,18 @@ protected static function fieldInfo($field_name = NULL) {
     return $fields_config;
   }
 
+  /**
+   * Check if the given name is an audience field.
+   *
+   * @param $field_name
+   *   The field name.
+   *
+   * @return bool
+   *   TRUE if the field name represent audience field.
+   */
+  public static function isAudienceField($field_name) {
```

Comment:
We already have a method for this that correctly checks the field type. See https://github.com/amitaibu/og/commit/3993f9a8c516e5e95678901ac966912e56465e18